lib/util/GraphicsUtil.js

```diff
@@ -23,9 +23,10 @@ export function getVisual(gfx) {
 /**
  * Returns the children for a given diagram element.
  *
- * @param {Snap<SVGElement>} gfx
- * @return {Snap<SVGElement>}
+ * @param {SVGElement} gfx
+ *
+ * @return {SVGElement}
  */
 export function getChildren(gfx) {
-  return gfx.parentNode.childNodes[1];
+  return domQuery('.djs-children', gfx.parentNode);
```

Comment:
What is this good for? Is it part of the fix?

If not, can we undo this change?